python/tool/sphinx/BUILD

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+load("@vaticle_dependencies_tool_docs//:requirements.bzl", docs_requirement = "requirement")
+
+py_binary(
+    name = "sphinx_runner",
+    srcs = [
+        "sphinx_html_builder_runner.py",
```

Comment:
can we just call the file `sphinx_runner.py`? Why do we need html and builder as well?

Reply:
We can. But sphinx can generate different formats of documentation, and we use only html.